src/alhambra/mixes.py

```diff
@@ -2097,6 +2119,118 @@ def tubes_markdown(self) -> str:
             entries.append(entry)
         return "\n".join(entries)
 
+    def instructions(
+        self,
+        plate_type: PlateType = PlateType.wells96,
+        validate: bool = True,
+        combine_plate_actions: bool = True,
+        well_marker: None | str | Callable[[str], str] = None,
+        title_level: Literal[1, 2, 3, 4, 5, 6] = 2,
+        warn_unsupported_title_format: bool = True,
+        tablefmt: str | TableFormat = "github",
```

Comment:
`Mix.table` uses "pipe" rather than "github".  They are similar, and both are supported by both Jupyter and pandoc, but "pipe" allows specification of column alignment.

Reply:
fixed (in this branch)